index.html

```diff
@@ -21,37 +21,49 @@
           <option value="LU">language of random</option>
         </select>
       </p>
+<!-- This is pretty unclear for a newbie (like me) .. should break it up and provide more detail (with comments)? -->
       <form action="/action_page.php">
-        <label for="fname">
+        <label for="inputtext">
           <span lang="en">Message:</span>
           <span lang="de">Nachricht:</span>
           <span lang="cn">讯息:</span>
           <span lang="es">Mensaje:</span>
           <span lang="LU">QWERTYUIOP:</span>
         </label>
-        <input type="text" id="fname" name="fname"><br><br>
-        <label for="sname">
-          <span lang="en">Secret:</span>
+<!--        <input type="inputtext" id="inputtext" name="inputtext"><br><br> -->
+        <textarea id="inputtext" name="inputtext" rows="4" cols="50">
+        Enter message to be de/encrypted.
```

Comment:
Nice :smile:

We need to think about translating that text too at some point. Maybe not in this PR but in a follow-up.